ansible_rulebook/conf.py

```diff
@@ -19,6 +19,7 @@ class _Settings:
     def __init__(self):
         self.identifier = str(uuid.uuid4())
         self.gc_after = 1000
+        self.execution_strategy = "sequential"
```

Comment:
I suggest to rename this variable as `default_execution_strategy` to avoid potential confusions.